### DISCUSSION TITLE
Rtmp broadcast stop automatically

### DISCUSSION THREAD

**Discussion by huangchukun** [mynaparrot/plugNmeet-server#192] [General]
I notice that the rtmp broadcast stopped streaming to CDN automatically after 6-7 minutes. Anyone knows why and how should I fix it?

**Comment by jibon57** [comment]
Which CDN provider are you using? We tested more than 1 hour for broadcasting using YouTube & Facebook without any problem.

**Reply by huangchukun** [reply]
I was using Alibaba Cloud. It seems that the ffmpeg output a high-bitrate stream (20000kbps). Any idea how could I change that parameter? Thanks

**Reply by jibon57** [reply]
You can try to play with those params:
https://github.com/mynaparrot/plugNmeet-recorder/blob/bf4c958d344242fbb6a2c2722e215c28b8b3da0b/src/utils/rtmpService.ts#L22
https://github.com/mynaparrot/plugNmeet-recorder/blob/bf4c958d344242fbb6a2c2722e215c28b8b3da0b/src/utils/rtmpService.ts#L28
